circleci/images/exttester/Dockerfile

```diff
@@ -90,8 +88,10 @@ RUN apt-get update \
     curl \
     debian-archive-keyring \
     gcc \
+    gdb \
     gnupg \
```

Comment:
git not needed here? Guess it is already in `buildpak-deps:bullseye` then?

Reply:
I guess so